api/app/services/api_documentation/dry_types_parser.rb

```diff
@@ -51,11 +55,21 @@ def merge_enum_values(hash, type)
         hash.merge(enum: type.values.map { |key, _| key })
       end
 
-      def merge_meta_values(hash, type)
-        filtered_meta = type.meta.select do |k, _|
+      def allowed_meta_values(type, allowed_swagger_meta_types)
+        type.meta.select do |k, _|
           allowed_swagger_meta_types.include? k
         end
-        hash.merge(filtered_meta)
+      end
+
+      def add_description_for_uniqueness(converted_hash)
+        converted_hash[:description] = (converted_hash[:description].to_s + " Must be unique.").strip
```

Comment:
I went with this implementation of string concatenation on the attribute description since it is the most concise and I am only combining two strings, but  if I need to do any more string building on the description in the future, I will be sure to change this to the array `.join(" ")` approach.